version/generate_version

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# generates the version of file based on current git version
+
+cd $GOPATH/src/github.com/contiv/netplugin/version
+
+USE_RELEASE=$1
+
+if [ -z "${USE_RELEASE}" ]
+then
+  file_name=version_default.go
+  UseVersionStr="// +build !use_version" 
+  useDefaultVersionString="False"
+else
+  file_name=version_release.go
+  UseVersionStr="// +build use_version" 
+  useDefaultVersionString="True"
+fi
+
+GitCommitStr=`git show | grep commit | cut -f 2 -d ' ' | cut -b 1-10`
+VersionStr=`cat CURRENT_VERSION`
+BuildTimeStr=`date`
+
+cat > $file_name << EOF
+$UseVersionStr
+// NOTE: this file is auto-generated by PR commits
+
+package version
+
+const usedDefaultVersion = "$useDefaultVersionString"
+
+// sample values are specified here
+const (
```

Comment:
@mapuri 

> minor: could we instantiate a Info struct here instead of independent consts?

I wanted to do that, unfortunately there is no const structure in Go from what I understand. Please let me know if you know about a way to do this.
@erikh, @shaleman, @unclejack - would you know?